app/Main.hs

```diff
@@ -111,22 +112,22 @@ run' opts =
              case checkModules ok of
                Left err -> fail err
                Right readyModules ->
-                   do _ <- T.forM (o_hsOutDir opts) $ \dir ->
-                          do T.putStrLn $
-                                  "Required Haskell library is "
-                                  <> li_name HS.libraryInfo <> "@" <> li_version HS.libraryInfo
-                             mapM_ (runner dir HS.makeModule HS.makeFileName) readyModules
-                      _ <- T.forM (o_elmOutDir opts) $ \dir ->
-                          do T.putStrLn $
-                                  "Required Elm library is "
-                                  <> li_name Elm.libraryInfo <> " version " <> li_version Elm.libraryInfo
-                             mapM_ (runner dir Elm.makeModule Elm.makeFileName) readyModules
-                      _ <- T.forM (o_psOutDir opts) $ \dir ->
-                          do T.putStrLn $
-                                  "Required PureScript library is "
-                                  <> li_name PS.libraryInfo <> " version " <> li_version PS.libraryInfo
-                             mapM_ (runner dir PS.makeModule PS.makeFileName) readyModules
-                      return ()
+                   do
+                     forM_ (o_hsOutDir opts) $ \dir -> do
```

Comment:
I would like to keep the coding style consistent, so the `do` should go to the next line